haproxy/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
I felt that HAProxy could start without any problem if this shell script was not used.
I thought it could be removed.

Reply:
This Shell is from the official container of HA-Proxy.
 I don't hope to change or remove this to keep compatibility.

Reply:
@takara9 
I understand the reason.
May I have the url where the shell script is distributed?

Reply:
Thx!, here is URL.
https://github.com/docker-library/haproxy/blob/master/2.6/docker-entrypoint.sh

Reply:
Thank you.
Looking at the repository, it is provided under GPL 2.0, so may I ask you to write the license in the file?
https://github.com/docker-library/haproxy/blob/master/LICENSE